# Message file which can be used to transfer a vector of circle data. For more info see VectorData.msg
# Date: 04-12-2018
# By: Giel Oomen

string vector_name		# Name of the vector
VectorData[] allHolesDataVec	# Vector of 'VectorData' structs (from VectorData.msg)

